Source/Libraries/GSF.COMTRADE/SampleRate.cs

```diff
@@ -75,11 +75,16 @@ public SampleRate(string lineImage, bool useRelaxedValidation = false)
         /// <summary>
         /// Converts <see cref="SampleRate"/> to its string format.
         /// </summary>
-        public override string ToString()
-        {
-            // samp,endsamp
-            return $"{Rate.ToString(CultureInfo.InvariantCulture)},{EndSample}";
-        }
+        public override string ToString() => // samp,endsamp
+            Format($"{Rate},{EndSample}");
+
+        #endregion
+
+        #region [ Static ]
+
+        // Static Methods
+        private static string Format(FormattableString formattableString) =>
```

Comment:
Huh - this is a new on me. Odd that an interpolated string can change its stripes in what looks to be post interpolation...

Reply:
Yeah, the compiler has to be interpreting the string differently depending on how it's used. Intellisense still reports the interpolated string as `System.String` on mouseover, but allows me to assign it to a variable of type `FormattableString`. It makes me a bit feel like I'm doing something wrong, but the compiler also doesn't let me assign any old string to a variable of type `FormattableString` so it must be fine so long as there's no compiler errors.